README.md

```diff
@@ -854,6 +854,7 @@ These colorschemes may not specialize in Tree-sitter directly but are written in
 - [chrisgrieser/nvim-alt-substitute](https://github.com/chrisgrieser/nvim-alt-substitute) - A substitute of vim's `:substitute` that uses Lua patterns instead of vim regex. Supports incremental preview.
 - [ckolkey/ts-node-action](https://github.com/ckolkey/ts-node-action) - A framework for executing functional transformations on Tree-sitter nodes - Has a lot of built-in actions for transforming text.
 - [tomiis4/hypersonic.nvim](https://github.com/tomiis4/hypersonic.nvim) - Provides explanation for RegExp.
+- [okuuva/auto-save.nvim](https://github.com/okuuva/auto-save.nvim) - Save your work as much as needed, without triggering external dependencies to much. Maintained fork of Pocco81/AutoSave.
```

Comment:
I absolutely would love to promote [deferred saving](https://github.com/okuuva/auto-save.nvim/issues/8) here since it's by far the best feature of the plugin imo but I'm not sure about the wording here. At first I thought you meant `noautocmd` but that's not in `main` yet.

How about "Automatically save your work after you stop editing it for a while" or something like that?

I would definitely keep the mention about being a maintained fork of Pocco81's work but I'd rename it to `Pocco81/auto-save.nvim`, both here and in the original. Since the original plugin was renamed as such before it went unmaintained.

Reply:
I like your suggestion but I think we can be a bit more colorful. I let ChatGPT generate some nice lines, based on yours, here are my favourites:

> Never lose your precious edits again - your changes are guarded and automatically preserved at just the right moment. No more frantic finger-dancing on the `:w` key. Maintained fork of `Pocco81/auto-save.nvim`.

> Automatically saves your work as much as needed and as little as possible. Customizable with smart defaults. Maintained fork of `Pocco81/auto-save.nvim`.

> Embrace productivity, embrace peace of mind - never lose your work, don't worry about external file watchers. Maintained fork of `Pocco81/auto-save.nvim`.

Any combination you'd like?
Otherwise I'll just go with yours :)

Reply:
I kinda like the second one, but use "often" and "seldom" instead of "much" and "little":
>Automatically saves your work as often as needed and as seldom as possible. Customizable with smart defaults. Maintained fork of Pocco81/auto-save.nvim

Maybe I'd also add a mention about skipping autocmds since that also got in:
>Automatically saves your work as often as needed and as seldom as possible, optionally skipping autocmds. Customizable with smart defaults. Maintained fork of Pocco81/auto-save.nvim

Reply:
I decided for the first one, the autocmd skipping is just a feature of many and falls under "Customizable".

Thanks for your help with this :)